zad5/app/views/tracks/index.html.erb

```diff
@@ -20,8 +20,8 @@
           </td>
           <td class="text-center"></td>
           <td class="text-right">
-            <%= link_to 'edit', '#', class: 'btn btn-warning' %>
-            <%= link_to 'remove', '#', class: 'btn btn-danger'  %>
+            <%= link_to 'edit', 'edit_track_path(track)', class: 'btn btn-warning' %>
+            <%= link_to 'remove', 'trck_path(track), method: :delete', class: 'btn btn-danger'  %>
```

Comment:
można dodać `data: { confirm: 'Are you sure?' }`, aby przypadkiem nie usunąć niczego 🙈